sql/testdata/datetime

```diff
@@ -42,6 +42,99 @@ statement error duplicate key value \(a\)=\(2015-08-30 03:34:45\.34567\+00:00\)
 INSERT INTO t VALUES
   ('2015-08-29 20:34:45.34567-07:00'::timestamp, '2015-08-31'::date, '35h2s'::interval)
 
+statement ok
+SET TIME ZONE 'Europe/Rome'
+
+query TTT
+SELECT * FROM t WHERE a = '2015-08-25 05:45:45.53453 CET'::timestamp
```

Comment:
`SELECT * FROM t WHERE a = $1`(and then checking for zero or one rows) is a convoluted way to test this. Just do `SELECT '2015-...'::timestamp`.